src/Microsoft.Identity.Web/Constants/LogMessages.cs

```diff
@@ -21,5 +21,7 @@ internal static class LogMessages
         public const string SessionCacheKeyNotFound = "Cache key {0} not found in session {1}. ";
         public const string SerializingSessionCache = "Serializing session {0}, cache key {1}. ";
         public const string ClearingSessionCache = "Clearing session {0}, cache key {1}. ";
+
+        public const string ErrorAcquiringTokenForOboForWebApi = "Error acquiring a token for obo for a web API - MsalUiRequiredException message is: {0} .";
```

Comment:
```suggestion
        public const string ErrorAcquiringTokenForOboForWebApi = "Error acquiring a token for On-Behalf-Of flow for a web API - MsalUiRequiredException message is: {0} .";
```

Reply:
i'm going to call it downstream web api instead, as OBO is not a flow, and it is confusing for a customer. thanks, peter. :)

---
In reply to: [464802916](https://github.com/AzureAD/microsoft-identity-web/pull/383#discussion_r464802916) [](ancestors = 464802916)

Reply:
Yea, downstream API makes sense.

Yea, idk I got the phrase from MS docs: https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow. :)